openApiDocs/beta/Applications.yml

```diff
@@ -536,6 +536,28 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  '/applications/{application-id}/appManagementPolicies/$count':
+    get:
+      tags:
+        - applications.appManagementPolicy
+      summary: Get the number of the resource
+      operationId: applications.appManagementPolicies_GetCount-3aad
```

Comment:
`$count` `Operation`s have a hash suffix. This will lead to us generating command names with the hash in them. In this case, the command will have a name like `Get-MgBetaApplicationAppManagementPolicyCount3Aad`.